index.js

```diff
@@ -1,4 +1,6 @@
 'use strict';
+var Intl = require('intl');
```

Comment:
note to myself: review `intl` library

Reply:
`intl` is a polyfill library for [ECMAScript Internationalization API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl). It was added to nodejs in later release versions.

Reply:
@sndsabin yeah the library has been considered as no longer maintained which is somewhat concerning. Also, last publish was 4 years ago. I wonder if most of the browsers already support the API as of now and if we should let the end users to add the polyfill as a pre-requisite on their own if they need to support this for older browsers. I believe Node had added this since 6.x, at least the basic support.

Reply:
@techgaun removed `intl` lib as suggested. Please review :)